From eca38c8097c337686ff5c77320739bd74f1de262 Mon Sep 17 00:00:00 2001
From: Kille135 <tyulkovvlad@gmail.com>
Date: Tue, 26 Jul 2022 16:30:20 +0300
Subject: [PATCH] sss

---
 class birds.java | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/class birds.java b/class birds.java
index fed8272..a89c520 100644
--- a/class birds.java	
+++ b/class birds.java	
@@ -8,7 +8,7 @@ class birds
     public static void main(String args[])
             throws IOException
     {
-        System.out.println("Сколько птичек?);
+        System.out.println("Сколько птиц?);
         BufferedReader reader = new BufferedReader(new InputStreamReader(System.in));
         Boolean maxbird;
         int bird = 1;
-- 
2.37.1.windows.1

